.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: node_js
+
+install:
+- cd extension; npm install; cd ..
+- cd frontend; npm install; cd ..
+
+env: # important! Without the top-level env, no job will be allowed to fail.
+  - JOB_DEFAULT=test
+
+matrix:
+  fast_finish: true
+  include:
+    - node_js: 8
+      env: TYPE=UNIT_CODESTYLE_DEPENDENCY
+      script:
```

Comment:
Also this looks overcomplicated. And since we don't have backend here, maybe would be a good idea to just use the travis config from other similar extensions like ext-upsell or ext-cliplister

Reply:
Unfamiliar with travis.yml file. I used boilerplate to generate and am using npm test to run jest. Not sure if I need to have that defined in my travis file